# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../arphound/arphound-1.3.2-CXX.patch
# Copyright (C) 2006 - 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -ru arphound-1.3.2-orig/Makefile arphound-1.3.2/Makefile
--- arphound-1.3.2-orig/Makefile	2006-11-26 17:38:16.000000000 +0100
+++ arphound-1.3.2/Makefile	2006-11-26 17:40:05.000000000 +0100
@@ -13,7 +13,7 @@
 	@cd shared && make
 
 arphound: binary 
-	$(CPP)  -o arphound $(OBJ_SRC:%=src/%.o) $(OBJ_SHARED:%=shared/%.o) -lpcap -lpthread
+	$(CXX)  -o arphound $(OBJ_SRC:%=src/%.o) $(OBJ_SHARED:%=shared/%.o) -lpcap -lpthread
 
 install:
 	@echo -n "Installing ArpHound... "
diff -ru arphound-1.3.2-orig/shared/Makefile arphound-1.3.2/shared/Makefile
--- arphound-1.3.2-orig/shared/Makefile	2006-11-26 17:38:16.000000000 +0100
+++ arphound-1.3.2/shared/Makefile	2006-11-26 17:40:34.000000000 +0100
@@ -9,7 +9,7 @@
 ob: $(OBJ_SHARED:=.o)
 
 .C.o:
-	$(CPP) $(C_FLAGS) -c $<
+	$(CXX) $(C_FLAGS) -c $<
 
 .c.o:
 	$(CC) $(C_FLAGS) -c $<
diff -ru arphound-1.3.2-orig/src/Makefile arphound-1.3.2/src/Makefile
--- arphound-1.3.2-orig/src/Makefile	2006-11-26 17:38:16.000000000 +0100
+++ arphound-1.3.2/src/Makefile	2006-11-26 17:40:21.000000000 +0100
@@ -11,7 +11,7 @@
 ob: $(OBJ_SRC:=.o)
 
 .C.o:
-	$(CPP) $(C_FLAGS) $(DEFINES) -c $<
+	$(CXX) $(C_FLAGS) $(DEFINES) -c $<
 
 .c.o:
 	$(CC) $(C_FLAGS) -c $<
